benchmark/paddle/SAC/README.md

```diff
@@ -0,0 +1,31 @@
+## Reproduce SAC with PARL
+Based on PARL, the SAC algorithm of deep reinforcement learning has been reproduced, reaching the same level of indicators as the paper in Mujoco benchmarks.
+
+> Paper: SAC in [Soft Actor-Critic: Off-Policy Maximum Entropy Deep Reinforcement Learning with a Stochastic Actor](https://arxiv.org/abs/1801.01290)
+
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
+
+### Benchmark result
+
+<img src=".benchmark/SAC_results.png" width = "800" height ="400" alt="SAC_results"/>
```

Comment:
Can you add the figures for other environments? 
There is only one environment here.

Reply:
done